src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/DeprecatedListWriter.java

```diff
@@ -90,7 +91,7 @@ protected void addExtraSection(DeprecatedAPIListBuilder list, Content content) {
                         ? contents.getContent("doclet.Deprecated_API_Checkbox_Other_Releases")
                         : Text.of(release);
                 HtmlId htmlId = HtmlId.of("release-" + releaseIndex);
-                tabs.add(HtmlTree.LABEL(htmlId.name(),
+                tabs.add(new HtmlTree(TagName.WBR)).add(HtmlTree.LABEL(htmlId.name(),
```

Comment:
Possible (future?) candidate for a static factory method.
Do we want to formally transition to always using static factory methods, and make the `new HtmlTree` constructors private?